{"metadata": {"dbt_schema_version": "https://schemas.getdbt.com/dbt/sources/v3.json", "dbt_version": "1.4.5", "generated_at": "2023-03-18T00:30:55.331577Z", "invocation_id": "e7782fac-432b-46b3-8fe2-791f7efad2e5", "env": {}}, "results": [{"unique_id": "source.dbtlearn.airbnb.reviews", "error": "Database Error in source reviews (models/sources.yml)\n  000904 (42000): SQL compilation error: error line 2 at position 10\n  invalid identifier 'DATA'", "status": "runtime error"}], "elapsed_time": 1.9807300567626953}